director/scripts/test/kinematic_test.py

```diff
@@ -30,21 +34,36 @@ def __init__(self, parent = None):
     def handle_thread(self):
         self.channel = "NETWORK_CASSIE_STATE_DISPATCHER"
```

Comment:
Everything except for the 
```
while true
            self.lcm.handle_timeout(100)
```
should probably be moved to the main thread (I know it was written like this in the test code I sent you)